src/lib/objects.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable object-shorthand */
+const createPerson = (name, age) => {
+  return {
+    name: name,
+    // eslint-disable-next-line object-shorthand
+    age: age
+  };
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  // eslint-disable-next-line no-prototype-builtins
+  return object.hasOwnProperty(property);
```

Comment:
As above, you can remove this rule as it isnt needed here. To get around the error, its telling you to use another method to access to objects property, which we can easily do using the square bracket notation instead:
```
  return object[property];
```